frontend/src/pages/SingleContract.tsx

```diff
@@ -1,22 +1,44 @@
 import ActivityFeed from 'components/activity-feed/ActivityFeed';
 import { ApproveJobForm } from 'components/single-contract/ApproveJobForm';
+import { CloseJobForm } from 'components/single-contract/CloseJobForm';
 import { CompleteJobForm } from 'components/single-contract/CompleteJobForm';
 import SingleContractData from 'components/single-contract/SingleContractData';
 import SingleContractHeading from 'components/single-contract/SingleContractHeading';
+import { useJob } from 'components/smart-contracts/hooks/useJob';
+import { useWallet } from 'components/wallet/useWallet';
 import { JobState } from 'enums/JobState';
-import React from 'react';
+import React, { useEffect } from 'react';
+import { addressesMatch } from 'utils/ethereum';
 import Loader from '../components/full-screen-loader/FullScreenLoader';
-import { useJob } from '../components/smart-contracts/hooks/useJob';
+import { useSingleContract } from '../components/single-contract/context/useSingleContract';
 
 const SingleContract: React.FC = () => {
   const { job, isLoading } = useJob();
 
+  const { account } = useWallet();
+
+  const { setSingleContract } = useSingleContract();
```

Comment:
Shouldn't need all this stuff anymore - just use `useJob`

Reply:
Added #108